recipes/gstreamer/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.8.0" %}
+{% set sha256 = "947a314a212b5d94985d89b43440dbe66b696e12bbdf9a2f78967b98d74abedc" %}
+
+package:
+  name: gstreamer
+  version: {{ version }}
+
+source:
+  fn: gstreamer-1.8.0.tar.xz
+  url: https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win or osx]
```

Comment:
I was just wondering what was the issue. I don't really want to get you down a rabbit hole trying to get this to work in `OS X`. (I did this with `harfbuzz` and friends and I realized why Mac people here ran from such a task!)